src/Spice86.Core/Emulator/Devices/Input/Keyboard/KeyboardPorts.cs

```diff
@@ -0,0 +1,22 @@
+namespace Spice86.Core.Emulator.Devices.Input.Keyboard; 
+
+/// <summary>
+/// IDs of keyboard ports for both read and write. <br/>
+/// https://www.win.tue.nl/~aeb/linux/kbd/scancodes-11.html
+/// </summary>
+public static class KeyboardPorts {
+    /// <summary>
+    /// Port used by the CPU to read the input buffer, or write to the output buffer, of the keyboard controller.
+    /// </summary>
+    public const byte IOPort = 0x60;
```

Comment:
How about calling this Data like they do here? https://bochs.sourceforge.io/techspec/PORTS.LST

Reply:
Done.

Reply:
Done.